documentation/glossary.md

```diff
@@ -0,0 +1,56 @@
+# Glossary of aviation terms
+*Openscope uses some terms specific to aviation, this glossary attempts to define them.*
+---
+## ILS
+*abbr. of* Instrument Landing System
+
+A radio antenna at the end of the runway, known as a ''localizer'', guides pilots down onto the runway for landing, via an instrument known as a *guide scope*.
+
+## Fix
+
+A pre-defined point that describes an aircraft's current position. A fix can be an airport (`EGKK`), a waypoint (`WILLO`), or other points, such as a VOR station (`GWC`).
+
+## Waypoint
+
+A fix that is used to aid in navigation, defined in a flight plan or route. Waypoints are assigned a 5-letter code, such as `WILLO` and `ABTUM`.
+
+## Flight Level
+*abbr.* FL
+
+A plane's altitude, in 100s of feet. For example, a plane at `FL350` is flying at 35,000ft.
+
+## SID
+*abbr. of* Standard Instrument Departure
+
+A route that an aircraft will follow to exit the airport's radar control area. Each SID begins at the airport and terminates at a fix, with waypoints in between.
+
+## STAR
+*abbr. of* Standard Terminal Arrival Route
+
+A route that an aircraft will follow to approach the airport, the inverse of a SID.
+
+## Airspace
+
+A pre-defined three-dimensional area where certain restrictions occur. For example, in Class A airspace, VFR aircraft are not permitted.
+
+## VFR
+*abbr. of* Visual Flight Rules
+
+VFR aircraft do not have instruments enabling them to navigate and land under ILS. As such, they rely on visual clues and ATC instruction to navigate and land.
```

Comment:
Nah, let's keep it. Might as well have information distinguising IFR/VFR. The VFR guys _may_ not have the fancy equipment, but just about all of them can shoot an ILS if they want to. It's as @indianbhaji says, just a set of rules they're following (VFR allows for more freedom and variation, provided they stay out of the way of stuff).

> A set of flight rules under which aircraft may fly visually, rather than depending completely on flight instruments. Because visibility is obviously required, VFR is only permitted when the cloud ceiling is at least 1,000 feet above ground level, and the visbility at least 3 statute miles, as well as remaining clear of clouds. Under VFR, aircraft do not have a flight plan filed with air traffic control, and therefore they may fly wherever they want, provided that they are talking to the right controllers when entering airspace of an elevated classification. They can therefore choose to spin in circles, fly along rivers and highways, fly direct to their destination, or follow a route just like an IFR aircraft would.